.github/release-please.yml

```diff
@@ -1,2 +0,0 @@
-releaseType: node
```

Comment:
this functionality was replaced with a [GitHub Action](https://github.com/GoogleCloudPlatform/release-please-action).